ci/caddy/Dockerfile

```diff
@@ -0,0 +1,9 @@
+# build Caddy with the Vultr DNS-01 challenge provider
+# see https://caddyserver.com/docs/modules/dns.providers.vultr
+FROM caddy:2.10-builder-alpine AS builder
+RUN xcaddy build --with github.com/caddy-dns/vultr@v0.0.0-20250723121531-55bf3e9768be
```

Comment:
Final note that we should bump the `caddy-dns/vultr@v...` version here once it's properly released - I'm tracking the relevant [issue](https://github.com/caddy-dns/vultr/issues/8).